covidmap/src/main/webapp/index.html

```diff
@@ -15,11 +15,62 @@
     <div id="info"></div>
 
     <div id="shared">
-        <div id="map"></div>
+        <div id="map">
+            <form class="search" action="">
+                    <label for"">News</label>
+                <input class="input" type="text" />
+                <input type="submit" />
+            </form>
+
+            <div class="container">
+                <ul class="news-list"></ul>
+            </div>
+        </div>
     </div>
 
     <div class="footer">
         <p>Footer/Contact</p>
     </div> 
+
+    <script>
+        const searchFrom = document.querySelector('.search');
+        const input = document.querySelector('.input');
+        const newsList = document.querySelector('.news-list');
+
+        searchFrom.addEventListener('submit', newsFetch)
+
+        function newsFetch(e) {
+            // checks if search field is empty
+            if (input.value == '') {
+                alert('Please enter search!')
+                return
+            }
+            
+            e.preventDefault()
+
+            const apiKey = 'nalhGwkCIzLTssWOSn8LbXWKZ4AhIHCW' // API KEY for NYTimes api
```

Comment:
I'm fine with this for the time being, but in general, hard coding API keys as magic constants is something we want to avoid. We should investigate what it looks like to load this from a config file.